packages/kbn-shared-ux-components/src/index.ts

```diff
@@ -58,6 +58,23 @@ export const LazyNoDataViews = React.lazy(() =>
  */
 export const NoDataViews = withSuspense(LazyNoDataViews);
 
+/**
+ * A pure `NoDataViews` component, with no services hooks. Consumers should use `React.Suspennse` or the
+ * `withSuspense` HOC to load this component.
+ */
+export const LazyNoDataViewsComponent = React.lazy(() =>
```

Comment:
I wonder if we shouldn't change the naming convention to `NoDataViewsComponentLazy` so the consumer sees the component grouped alphabetically.

Reply:
I like the idea! Will do it in a follow-up PR

Reply:
I'm considering a codemod to make them all `MyComponent_lazy`... just forgot to mention it during our last meeting. 🤷🏻